src/components/common/ProductBox/ProductBox.module.scss

```diff
@@ -68,11 +70,20 @@
       transform: translateX(-50%);
     }
   }
-
+  .price div{
+    transition: 0.4s ease-in;
```

Comment:
Could use class instead of selecting by tagname? You can add new class to component view if needed.